### ISSUE TITLE
visage-app wont start

### ISSUE THREAD

**Issue by ghost** [auxesis/visage#142]
after installing got error

visage-app start
/var/lib/gems/1.9.1/gems/visage-app-2.1.0/bin/visage-app:6:in `gethostbyname': getaddrinfo: Name or service not known (SocketError)
        from /var/lib/gems/1.9.1/gems/visage-app-2.1.0/bin/visage-app:6:in`<top (required)>'
        from /usr/local/bin/visage-app:23:in `load'
        from /usr/local/bin/visage-app:23:in`<main>'

My installed components:

gem list

**\* LOCAL GEMS ***

errand (0.7.3)
haml (3.1.4)
rack (1.5.2)
rack-protection (1.5.1)
sinatra (1.3.2)
tilt (1.3.3)
visage-app (2.1.0)
yajl-ruby (1.1.0)

And ruby packages:

librrd-ruby 1.4.7-2
librrd-ruby1.8 1.4.7-2
libruby1.8 1.8.7.358-7
libruby1.9.1 1.9.3.194-8.1+deb7u1
ruby 1:1.9.3
ruby-dev 1:1.9.3
ruby-passenger 3.0.13debian-1
ruby-rack 1.4.1-2.1
ruby1.8 1.8.7.358-7
ruby1.8-dev 1.8.7.358-7
ruby1.9.1 1.9.3.194-8.1+deb7u1
ruby1.9.1-dev 1.9.3.194-8.1+deb7u1
rubygems 1.8.24-1

**Comment by jessereynolds** [comment]
Yep, I see the same thing with visage 2.1.0 on Ubuntu 12.04 when running under Apache, after following the instructions in the readme (with "lib/lib/" -> "lib/" fix). When you load the web interface you get a Phusion Passenger error as described above - the webserver does not have permissions to modify the profiles. 

Workaround:

```
sudo chown nobody /var/lib/gems/1.8/gems/visage-app-2.1.0/lib/visage-app/config/profiles.yaml
```
